### DISCUSSION TITLE
identify available package options in iso

### DISCUSSION THREAD

**Discussion by jshen28** [rhinstaller/anaconda#3601] [Q&A] [answered]
Hello,

  I am trying to find available packages for kickstart file. Right now to identify correct package setting I have to provision a virtual machine and use it. My question is is it possible to identify all available package in iso beforehand.. 

FYI for packages I mean something like @^minimal-environment

Best,
Norman

**Comment by slayercat** [comment, marked as answer]
I'm suggest to use `dnf groupinstall --downloadonly`

for details, may be this could help: https://flylib.com/books/en/1.362.1.42/1/
